CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - test for biceps:5-4-7_3
 - test for biceps:5-4-7_5
 - test for biceps:5-4-7_7
+- test for biceps:5-4-7_11
+- support for SystemErrorReport to test case glue:R0036_0
```

Comment:
not part of this pr

Reply:
The last line was merged in from main. The only thing I did was to reorder the lines.

Reply:
I merged in the master again. So be careful, the diff might look weird again...